pages/common/spf.md

```diff
@@ -0,0 +1,32 @@
+# spf
+
+> The superfile – Modern terminal file manager.
+> More information: <https://github.com/yorukot/superfile>.
+
+- Launch `spf` with a specific path:
+
+`spf {{/path/to/start}}`
+
+- Launch `spf` with multiple paths:
+
+`spf {{/path/to/start1}} {{/path/to/start2}}`
+
+- Fix hotkey settings by appending missing keys:
+
+`spf --fix-hotkeys`
```

Comment:
In the CLI usage documentation I would have liked to check if there are short options of these.

Reply:
```suggestion
`spf {{[-fh|--fix-hotkeys]}}`
```

Reply:
Add the help page in the front page readme.md so that it's easily accessible withoyt needing to install the program.